### ISSUE TITLE
About the relationship between camera2lidar and lidar2image

### ISSUE THREAD

**Issue by qq160816** [NVIDIA-AI-IOT/Lidar_AI_Solution#194]
Hi：
Excuse me.
Is the relationship between camera2lidar and lidar2image inverse?

**Comment by YueSun0609** [comment]
> Hi： Excuse me. Is the relationship between camera2lidar and lidar2image inverse?

No, camera is not the same as image, you can search them online.
